provisioner/ansible/provisioner.go

```diff
@@ -63,10 +63,14 @@ type Config struct {
 	UseSFTP              bool     `mapstructure:"use_sftp"`
 	InventoryDirectory   string   `mapstructure:"inventory_directory"`
 	InventoryFile        string   `mapstructure:"inventory_file"`
+	KeepInventoryFile    bool     `mapstructure:"keep_inventory_file"`
```

Comment:
I found this helpful when debugging, and realized it would be helpful to users who want to debug, too, so I turned it into an option instead of just commenting out the code while I worked on it. It probably should be its own PR but it's such a small element of the overall diff I figured it doesn't add that much extra complexity to review.

Reply:
It's a nice touch for sure. When I first reviewed the PR I thought that it would be helpful for users if they run into issues debugging. Nicely done :heavy_check_mark: